### PR TITLE
Added 'is a Date, Is a Leap Year, Is Before, Subtract, Day of Year'

### PR DESCRIPTION
**Pull request by ODudek**
I also added `is a Date`, `Is a Leap Year`, `Is Before`, `Subtract`, Partial `Day of Year` - added `get` but i've got some issue with `set`

### CODE REVIEW

**Comment by ODudek** [comment]
Weird... on my local machine all test cases are passing, but on travis doesn't.. anyway i removed unvalid test and i will work on it

File: __tests__/index.js [hunk 1]
```diff
@@ -226,10 +235,22 @@ describe('Manipulate', () => {
 
   it('Subtract', () => {
     const m = moment(time).subtract(7, 'days');
+    const n = new Date(
```
Comment (stevemao) [inline __tests__/index.js]: I'd rather do something like `Date(time).getTime() - /* whatever in milliseconds */`

File: README.md [hunk 2]
```diff
@@ -519,6 +529,18 @@ Subtract the specified number of days from the given date.
 moment().subtract(7, 'days');
 // => "2018-09-02T09:12:49.695Z"
 
+// Native
+new Date(
+  new Date(time).getFullYear(),
```
[the remark this answers is not in the document]
Comment (ODudek) [reply README.md]: I coppied from test and forgot to remove, sorry about that

File: __tests__/index.js (hunk 1 above)
Comment (ODudek) [reply __tests__/index.js]: Thanks, replaced

**Pull request merged**